public/app/core/services/echo/backends/analytics/FullstoryBackend.ts

```diff
@@ -0,0 +1,22 @@
+import { init } from '@fullstory/browser';
+
+import { EchoBackend, EchoEventType, FullstoryEchoEvent } from '@grafana/runtime';
+
+export interface FullstoryBackendOptions {
+  orgId: string;
+  devMode?: boolean;
+}
+
+export class FullstoryBackend implements EchoBackend<FullstoryEchoEvent, FullstoryBackendOptions> {
+  supportedEvents = [EchoEventType.Fullstory];
+
+  constructor(public options: FullstoryBackendOptions) {
+    init({ orgId: options.orgId, devMode: options.devMode ?? false, debug: options.devMode ?? false });
+  }
+
+  // Not using custom events, Fullstory track every interaction automatically
+  addEvent = (e: FullstoryEchoEvent) => {};
+
+  // Not using Echo buffering
+  flush = () => {};
+}
```

Comment:
Not sure EchoBackend is necessary here given we do not interact with the that Fullstory API at all, other than initialization. But it may be worth keeping it like this to keep these kinds of integration within the same bucket.

Reply:
Good point @dprokop! I did it for two reasons:
* Keep all the integrations in the same bucket as you mentioned :) 
* For now we won't send custom events but the SDK allows that. Probably, to link Rudderstack with FS we will do it at some point.